lib/logstash/codecs/netflow.rb

```diff
@@ -33,13 +33,17 @@ class LogStash::Codecs::Netflow < LogStash::Codecs::Base
   # See <https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/lib/logstash/codecs/netflow/netflow.yaml> for the base set.
   config :definitions, :validate => :path
 
+  NETFLOW5_FIELDS = ['version', 'flow_seq_num', 'engine_type', 'engine_id', 'sampling_algorithm', 'sampling_interval', 'flow_records']
+  NETFLOW9_FIELDS = ['version', 'flow_seq_num']
+  SWITCHED = /_switched$/
+  FLOWSET_ID = "flowset_id"
+
   public
```

Comment:
Even if it's a common pattern for all logstash, I found it very strange, we don't need to make methods public if there is no need for it, like in this case (they are by default). Is there any other reason that I might be missing to keep that ?